CHANGELOG.md

```diff
@@ -8,6 +8,13 @@ _**For better traceability add the corresponding GitHub issue number in each cha
 
 ## [Unreleased]
 
+## Added
+
+- Added endpoint for dedicated removal of policy from BPNL. #559 
+
+
+
+
 ## [5.1.1] - 2024-05-08
```

Comment:
```suggestion
## Added

- Added endpoint for dedicated removal of policy from BPNL. #559 


## [5.1.1] - 2024-05-08
```

Please leave only 2 newlines between each version. This avoids conflicts in the CHANGELOG